### DISCUSSION TITLE
Export your game without editors, and then publish it to app stores (WIP).

### DISCUSSION THREAD

**Discussion by Darenn** [nesbox/TIC-80#1836] [Q&A] [answered]
Hi !
Just wanted to know if it's still something in the works ?
**Export your game without editors, and then publish it to app stores (WIP).**

A fair number of testers escaped my game by accident and got lost in the editors.
While it's nice to keep it on the tic website so that everyone can see the code and play with it, I think it's best to hide this on other platforms like itch.io.

**Comment by nesbox** [comment, marked as answer]
Export without editors has been released since **0.90**, pls use `export <win linux rpi mac html> <game> alone=1` command.

**Reply by Darenn** [reply]
That's great thank you !
I don't think it is mentioned anywhere (sorry I this was the case).

I've made a pull request on the readme and made a change on the commands wiki page.

https://github.com/nesbox/TIC-80/wiki/Console

Please feel free to discard these if not appropriate. 

Have a nice day!

**Comment by erwingame** [comment]
Thank you

**Comment by dobkun** [comment]
so you literally paywalled a good command?
![New_Level_Retard](https://github.com/user-attachments/assets/52cfd14a-22c2-4222-bef7-e2687c49d6ae)

**Reply by Miguel-hrvs** [reply]
Lmao there's instructions about how you can get the pro version for free.
